index.html

```diff
@@ -1420,183 +1419,189 @@ <h5>
+            <p data-lang="zh-hans"><span class="leadin">间隔号</span></p>
             <p data-lang="zh-hant"><span class="leadin">間隔號</span></p>
-            <p data-lang="en"><span class="uname">U+00B7 MIDDLE DOT</span> [·], also known as interpoint, middot or centered dot, is a punctuation mark consisting of a vertically-centered dot, and is used to separate the first name and family name in names translated from a foreign language, or minority groups' names. It is also used with double quotation marks to separate chapters, articles and volumes in publications.</p>
-            <p data-lang="zh-hans" class="checkme">间隔号为 <span class="uname" translate="no">U+00B7 MIDDLE DOT</span> [·]。用于书名号乙式(双书名号)分隔篇、章、卷，以及外国人名中文译名、少数民族音译名，分隔姓名使用。</p>
-            <p data-lang="zh-hant">間隔號為<span class="uname">U+00B7 MIDDLE DOT</span> [·]。用於書名號乙式（雙書名號）分隔篇、章、卷，以及外國人名中文譯名、少數民族音譯名，分隔姓名使用。</p>
-            <p data-lang="en">Middle dots apply to Chinese only. When a translated foreign name contains a Latin character, the full stop should be used rather than the middle dot. For example, <span lang="zh-hant">「比尔·盖茨」</span> but <span lang="zh-hant">「B. 盖茨」</span>.</p>
-            <p data-lang="zh-hans" class="checkme">间隔号为中文标点，当外语姓名包含西文首字母缩写作为单节时，原则上西文字母后面应该使用西文句点，如“比尔·盖茨”、“B. 盖茨”。</p>
-            <p data-lang="zh-hant">间隔号为中文标点，当外语姓名包含西文首字母缩写作为单节时，原则上西文字母后面应该使用西文句点，如「比尔·盖茨」「B. 盖茨」。</p>
-            <p data-lang="en">Usage of middle dots differs between Traditional Chinese and Simplified Chinese. In principle, the middle dot, either in vertical writing mode or horizontal writing mode, should have the same dimensions as a character; however, in Simplified Chinese, the middle dot sometimes has half the width of a character when it is used to separate the month and the date, e.g. 9·11. </p>
-            <p data-lang="zh-hans" class="checkme">间隔号在繁、简中文间的用法有所不同，原则上繁体中文无论直排或横排，都占用一个汉字的大小；简体中文用于分隔日期（如 9·11）等情况时，可用半个汉字的大小。</p>
+
+            <p data-lang="en">Interpuncts <span class="uname">U+00B7 MIDDLE DOT</span> [·], also known as interpoints, middots or centered dots, are punctuation marks consisting of a vertically-centered dot, and is used to separate the first name and family name in names translated from foreign languages or minorities. They are also used with book title marks to separate chapters, articles and volumes in publications.</p>
+            <p data-lang="zh-hans">间隔号为<span class="uname" translate="no">U+00B7 MIDDLE DOT</span> [·]。用于双书名号（书名号乙式）分隔篇、章、卷，以及外国人名中文译名、少数民族音译名，分隔姓名使用。</p>
+            <p data-lang="zh-hant">間隔號為<span class="uname" translate="no">U+00B7 MIDDLE DOT</span> [·]。用於書名號乙式（雙書名號）分隔篇、章、卷，以及外國人名中文譯名、少數民族音譯名，分隔姓名使用。</p>
+            <p data-lang="en">Interpuncts apply to Chinese only. When a translated foreign name contains a Latin character, a western period should be used rather than a interpunct. For example, <span lang="zh-hant">比尔·盖茨</span> but <span lang="zh-hant">B. 盖茨</span>.</p>
```

Comment:
These examples don't look right, because no chinese font is applied to middle dot for the hant samples. The chinese characters are using a fallback font.  I'll raise a new issue about this.

Reply:
OK